composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^7.2.5 || ^8.0",
+        "php": "^7.4 || ^8.0",
```

Comment:
I think we should go back to not claiming we support future versions of PHP: `^7.4 || ^8.0 <8.2`.

Reply:
i don't think it would work as ppl with 8.2 would anyway install Fixer 3.4 which allows ^8 and fail on runcheck.

anyway, let's move the discussion outside of scope of this PR.

Reply:
I agree we should go back to not stating we support any (minor or major) versions of PHP we don't know we do (like future releases), but I also agree that it doesn't have to be on PR

Reply:
we don't claim the support. we moved the PHP version check to runtime, as composer-based check is not enough, like ppl are downloading phar file manually (or via phive) and basically skip potential composer-based check